tests/generator/test_auth.py

```diff
@@ -225,6 +225,21 @@ def test_auth_wired(self):
             self.assertEqual(stat.S_IMODE(os.fstat(f.fileno()).st_mode), 0o600)
         self.assertTrue(os.path.isfile(os.path.join(
             self.workdir.name, 'run/systemd/system/netplan-wpa-eth0.service')))
+
+        with open(os.path.join(self.workdir.name, 'run/systemd/system/netplan-wpa-eth0.service')) as f:
+            self.assertEqual(f.read(), '''[Unit]
+Description=WPA supplicant for netplan eth0
+DefaultDependencies=no
+Requires=sys-subsystem-net-devices-eth0.device
+After=sys-subsystem-net-devices-eth0.device
+Before=network.target
+Wants=network.target
+
+[Service]
+Type=simple
+ExecStart=/sbin/wpa_supplicant -c /run/netplan/wpa-eth0.conf -ieth0 -Dwired
```

Comment:
We should probably move this into a template string (e.g. `SD_WPA` in `tests/generator/base.by`), to combine the common pattern (filling it only with the relevant interface name and `ExecStart=` line.

This could then be reused in `test_wifis.py` as well, so we do not repeat ourselves too much.

Reply:
Ah-ha, good idea. I will prepare an updated commit. Thanks!